### DISCUSSION TITLE
Add the "always on top" toggle button back to the top bar

### DISCUSSION THREAD

**Discussion by grantholle** [spatie/ray#609] [Ideas]
On the previous version, the toggle to keep the window always on top was in the top bar of the window. I feel like there's enough real estate to keep it here, rather than to hunt in the preferences to toggle that option.

**Comment by AdrianMrn** [comment]
We've replaced it with a shortcut that is unset by default, you can set the shortcut in the preferences by "global shortcuts". You can also find the option to toggle it in the "window" menu at the top.
